src/mdbExtensionController.ts

```diff
@@ -422,10 +438,30 @@ export default class MDBExtensionController implements vscode.Disposable {
       name: 'MongoDB Shell',
       env: { MDB_CONNECTION_STRING: mdbConnectionString }
     });
+
+    if (this.isSslConnection(activeConnectionModel)) {
+      mdbTlsOptionsString =
+        '--tls --tlsAllowInvalidCertificates --tlsAllowInvalidHostnames';
```

Comment:
Our connection model also disables them: https://github.com/mongodb-js/connection-model/blob/master/lib/model.js#L634-L637

Reply:
TODO: check sslMethod to enable validation for some types of connections.